README.md

```diff
@@ -11,7 +11,8 @@
 ## Table of Contents
 
 - [CONTRIBUTING](./CONTRIBUTING.md)
-- [CI](./CI.md)
+- [GitHub Actions CI](./CI.md)
```

Comment:
nit: should we rename CI.md to github-actions-ci.md?

Reply:
I have a branch that moves all of these markdown files underneath `docs/`(since right now there spread all over). Id rather clean that up there if thats ok since well be moving it anyway.